tensorflow/python/estimator/estimator.py

```diff
@@ -1546,9 +1546,8 @@ def maybe_overwrite_model_dir_and_session_config(config, model_dir):
           "`model_dir` are set both in constructor and `RunConfig`, but with "
           "different values. In constructor: '{}', in `RunConfig`: "
           "'{}' ".format(model_dir, config.model_dir))
-    if model_dir:
-      config = run_config.RunConfig.replace(config, model_dir=model_dir)
-  if getattr(config, 'model_dir', None) is None:
+    config = run_config.RunConfig.replace(config, model_dir=model_dir)
```

Comment:
Can you undo this change? I was under the impression this CL was doc updates only.